functions/users.py

```diff
@@ -14,3 +13,23 @@ def disable_inactive_users(*args):
         for u in disabled:
             print(f"Disabled inactive: {u[0]}")
         print("done.")
+
+
+def refresh_download_permissions(*args):
+    """
+    Extend the expiry date for GCS download permissions belonging to users
+    who accessed the system today. If we don't do this, users whose accounts
+    are still active might lose GCS download permission prematurely.
+    """
+    active_today = lambda q: q.filter(
+        Users._accessed > datetime.today() - timedelta(days=1)
+    )
+    with sqlalchemy_session() as session:
+        active_users = Users.list(
+            page_size=Users.count(session=session),
```

Comment:
not sure why you would want for potentially unbounded `page_size` here?

Reply:
Good point - the same `active_today` filter should be applied to this query.